src/js/script.js

```diff
@@ -0,0 +1,56 @@
+let myLibrary = [];
+
+function Book(title, author, numPages, read) {
+  this.title = title;
+  this.author = author;
+  this.numPages = numPages;
+  this.read = read;
+}
+
+function addBookToLibrary() {
+  let title = document.querySelector("#title").value;
+  let author = document.querySelector("#author").value;
+  let numPages = document.querySelector("#numPages").value;
+  let read = document.querySelector("#read").value;
+
+  let newBook = new Book(title, author, numPages, read);
+  myLibrary.push(newBook);
+  render();
+  document.getElementsByClassName("input")[0].style.display = "none"; 
```

Comment:
The `addBookToLibrary` logic should be free of any `DOM` related code. Refactor your code to ensure all `DOM` tasks are done in the `render` function.